src/libs/ReportActionsUtils.ts

```diff
@@ -1764,7 +1764,7 @@ function getCardIssuedMessage(reportAction: OnyxEntry<ReportAction>, shouldRende
 
     const isAssigneeCurrentUser = currentUserAccountID === assigneeAccountID;
 
-    const shouldShowAddMissingDetailsButton = reportAction?.actionName === CONST.REPORT.ACTIONS.TYPE.CARD_MISSING_ADDRESS && missingDetails && isAssigneeCurrentUser;
+    const shouldShowAddMissingDetailsButton = !isAssigneeCurrentUser || (reportAction?.actionName === CONST.REPORT.ACTIONS.TYPE.CARD_MISSING_ADDRESS && missingDetails);
```

Comment:
This is confusing:

> if you are not the card assignee, you should not see the button. 

> if you are the assignee and you have all the personal details, you should not see the button

The first boolean should not be negated, right?
@shubham1206agra @allgandalf There are no recordings either to show testing

Reply:
ok, can you update it to what the variable represents actually then? Thank you